### ISSUE TITLE
Wrong output with classes and ternary operator

### ISSUE THREAD

**Issue by ACXgit** [tivac/mopt#57]
Hello, I noticed a weird behaviour in my app after running mopt, which seems to involve classes determined by ternary operators. Here's a quick example to reproduce the issue:

Source code:
```
const root = document.body;
const states = [true, false, null];
const state = states[Math.floor(Math.random() * states.length)];

m.render(root, m('span', {
        className: state === true ? 'text-success' : state === false ? 'text-danger' : 'text-muted'
}, 'status'));
```

Transformed result:
```
var root = document.body;
var states = [true, false, null];
var state = states[Math.floor(Math.random() * states.length)];

m.render(root, m.vnode('span', undefined, undefined, undefined, 'status', undefined));
```

**Comment by tivac** [comment]
I see the issue. I wrote some code that tries to combine any doubled-up instances of `class`/`className` props (from selector + attrs) and it makes some real bad assumptions. Working on a fix.

**Comment by tivac** [comment]
@ACXgit This is fixed in `mopt@5.0.2` which should be on npm shortly. Sorry about that!

**Comment by ACXgit** [comment]
No need to apologize :) I just tested v5.0.2 and it solved that specific issue, at least where it was previously broken in my app 👍
